deployment/HDI-Spark/Notebooks/README.md

```diff
@@ -0,0 +1,116 @@
+
+After you see green check-marks next to the status of both script actions, you can move to the next step.
+
+## Step 3. Start Livy Server
+Please follow the above [Step 1](#step-1-stop-livy-server) to **Start** Livy for Spark2 Server.
+
+## Step 4. Set up Spark Default Configs
+4.1 From the portal, select **Overview**, and then select **Ambari home**. If prompted, enter the cluster login credentials for the cluster.
+
+4.2 Select **Spark2**, and then Select **CONFIGS**, then click on **Custom spark2-defaults**.
+
+<img src="../../../docs/img/HDI-Spark-Notebooks/spark-configs.png" alt="SetConfigsImage" width="800"/>
+
+4.3 Click on **Add Property...** to add Spark default settings.
+You will add a total of 4 individual properties. Add them 1 one at a time, all using **TEXT** Property Type in Single property add mode. Make sure you don’t have any extra spaces before or after any of the keys/values.
+- Property 1:
+    - Key:&ensp;&ensp;`spark.dotnet.shell.command`
+    - Value: `/usr/share/dotnet-tools/dotnet-try,kernel-server,--default-kernel,csharp`
+- Property 2:
+    - Key:&ensp;&ensp;`spark.dotnet.packages`
+    - Value: `["nuget: Microsoft.Spark, 0.5.0", "nuget: Microsoft.Spark.Extensions.Delta, 0.5.0"]`
+- Property 3:
+    - Key:&ensp;&ensp;`spark.dotnet.version`
+    - Value: `0.5.0`
+- Property 4:
+    - Key:&ensp;&ensp;`spark.dotnet.interpreter`
+    - Value: `try`
+
+For example, the following captures the setting for adding property 1:
+
+<img src="../../../docs/img/HDI-Spark-Notebooks/add-sparkconfig.png" alt="SetConfigsImage" width="800"/>
+
+After adding those 4 properties, select **SAVE**. If you see a warning screen of config recommendations, select **PROCEED ANYWAY**.
+
+4.4 Restart affected components.
+After adding the new properties, you need to restart components that were affected by the changes. At the top, select **RESTART**, and then **Restart All Affected** from the drop-down.
+
+<img src="../../../docs/img/HDI-Spark-Notebooks/restart-affected.png" alt="SetConfigsImage" width="800"/>
+
+When prompted, select **CONFIRM RESTART ALL** to continue, then click **OK** to finish.
+
+# Submit Jobs through Jupyter Notebook
+After finishing previous steps, now you can submit your jobs through Jupyter Notebook!
+1. Create a new Spark .NET notebook
+
+[Launch Jupyter notebook](https://docs.microsoft.com/en-us/azure/hdinsight/spark/apache-spark-jupyter-spark-sql-use-portal#create-a-jupyter-notebook) from your HDI cluster in the Azure Portal, then select **New** > **Spark .Net** to create a notebook.
+
+<img src="../../../docs/img/HDI-Spark-Notebooks/create-sparkdotnet-notebook.png" alt="JupyterNotebookImage" width="800"/>
+
+2. Submit Jobs using Spark .Net
+
+> **Note:** After the kernel is ready, then you can start submitting jobs. The following shows a sample to submit jobs through notebook.
+- Start a spark session
+<img src="../../../docs/img/HDI-Spark-Notebooks/start-sparksession.png" alt="SubmitSparkJobImage" width="800"/>
```

Comment:
Change Spark .NET in Jupyter to .NET Spark (C#) to be consistent with Azure Synapse Analytics.

Reply:
Changed to  .NET Spark (C#) to be consistent with Azure Synapse Analytics.